storybook/main.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
 		'@storybook/addon-a11y',
 		'@storybook/addon-toolbars',
 		'@storybook/addon-actions',
+		'@storybook/addon-webpack5-compiler-babel',
```

Comment:
Needed in order to opt-in to using Babel explicitly, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-babelcore-and-babel-loader-from-storybookbuilder-webpack5